src/docs/product/relay/getting-started.mdx

```diff
@@ -66,7 +74,18 @@ _"create custom config"_ and customizing these parameters:
 Settings are recorded in `.relay/config.yml`. Note that all configuration values
 are optional and default to these settings:
 
-```yaml
+```yaml {tabTitle:Run in Docker}
```

Comment:
_nit_:

```suggestion
```yaml {tabTitle:Configuration for Docker}
```

Reply:
Looks like if we do this, tab switching will not be synced across the page :/